src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -388,7 +388,7 @@ export default class TutorialView {
         this.tutorial_step({
             title: 'Fixes',
```

Comment:
I'll defer to @erikquinn on this one.  I'm not sure the `route` command should be included in the tutorial.  we already have `pd` which does basically the same thing.  having both may be confusing to some users.